src/subscript/welltest_dpds/welltest_dpds.py

```diff
@@ -126,6 +122,17 @@ def get_parser():
         default="OIL",
         required=False,
     )
+    parser.add_argument(
+        "--genobs_resultfile",
+        type=str,
+        help=(
+            "File with welltest results used to define time steps, "
+            "typically exported from Saphir. If present, additional files "
+            "to be used as GEN_OBS in ERT are produced"
+        ),
+        default=None,
+        required=False,
```

Comment:
`required=False` is implicit for  `--`-options

Reply:
Fixed